### PR TITLE
Rework backstage for mobile

### PR DESCRIPTION
**Pull request by ParadoxARG**
## Purpose

* Create new buttons for mobile to show and hide the backstage
* Hide all the elements except the backstage when clicking on `show backstage` button click
* Hide the backstage and make all the elements visible again on `hide backstage` button click
* Add listener on screen size change and change the context according to the changes

_Small screen - Home_
image

_Small screen - Backstage_
image

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe: UI
```

### CODE REVIEW

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js [hunk 1]
```diff
@@ -143,6 +155,75 @@ document.addEventListener("DOMContentLoaded", function () {
...
+function showPageContent(){
+    console.log('show');
+    navbar.style.display = 'flex';
+    articleContainer.style.display = 'block';
+    graphContainer.style.display = 'flex';
+}
+
+// Makes the page content visible except for the backstage which will remain unchanged
+function hidePageContent(){
+    console.log('hide');
```
Comment (nbellocam) [inline demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: Remove console logs :)

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js [hunk 2]
```diff
@@ -143,6 +155,75 @@ document.addEventListener("DOMContentLoaded", function () {
...
+
+function setSmallLayoutConfiguration(){
+    if(backstage.classList.contains('show')){
+        hidePageContent();
+    }
+}
+
+// Hides the page content except for the backstage
+function showPageContent(){
+    console.log('show');
```
Comment (nbellocam) [inline demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: Remove console logs :)

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js [hunk 3]
```diff
@@ -143,6 +155,75 @@ document.addEventListener("DOMContentLoaded", function () {
...
+        if(currentSize == SCREEN_SIZE_SMALL){
+            currentSize = SCREEN_SIZE_BIG;
+            console.log("Window size is now BIG");
+            setBigLayoutConfiguration();
+        }
+    } else {
+        //console.log("Window size is now SMALL");
+        if(currentSize == SCREEN_SIZE_BIG){
+            currentSize = SCREEN_SIZE_SMALL;
+            console.log("Window size is now SMALL");
```
Comment (nbellocam) [inline demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: Remove console logs :)

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js [hunk 4]
```diff
@@ -143,6 +155,75 @@ document.addEventListener("DOMContentLoaded", function () {
...
+   console.log("Window detected as SMALL");
+   currentSize = SCREEN_SIZE_SMALL;
+}
+
+$(window).resize(function () {
+    if (jQuery(window).innerWidth() > mobileSize) {
+        //console.log("Window size is now BIG");
+        if(currentSize == SCREEN_SIZE_SMALL){
+            currentSize = SCREEN_SIZE_BIG;
+            console.log("Window size is now BIG");
```
Comment (nbellocam) [inline demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: Remove console logs :)

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js [hunk 5]
```diff
@@ -143,6 +155,75 @@ document.addEventListener("DOMContentLoaded", function () {
...
+const SCREEN_SIZE_SMALL = 0;
+const SCREEN_SIZE_BIG = 1;
+const mobileSize = 991;
+
+if ($(window).width() > mobileSize) {
+   console.log("Window detected as BIG");
+   currentSize = SCREEN_SIZE_BIG;
+}
+else {
+   console.log("Window detected as SMALL");
```
Comment (nbellocam) [inline demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: Remove console logs :)

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js [hunk 6]
```diff
@@ -143,6 +155,75 @@ document.addEventListener("DOMContentLoaded", function () {
...
+const backstage = document.getElementById('collapseBackstage');
+const backstageBtn = document.getElementById("backstage-btn");
+
+let currentSize;
+const SCREEN_SIZE_SMALL = 0;
+const SCREEN_SIZE_BIG = 1;
+const mobileSize = 991;
+
+if ($(window).width() > mobileSize) {
+   console.log("Window detected as BIG");
```
Comment (nbellocam) [inline demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: Remove console logs :)

**Review by nbellocam — CHANGES_REQUESTED** [review]

File: demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js (hunk 6 above)
Comment (ParadoxARG) [reply demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/wwwroot/js/demo.js]: 🌮 Done

**Pull request merged**